v/vanilla_bean/parameters.vh

```diff
@@ -203,6 +203,7 @@
 
 localparam RV32_instr_width_gp    = 32;
 localparam RV32_reg_data_width_gp = 32;
+localparam RV32_reg_els_gp        = 32;
```

Comment:
Is this just the number of RISC-V registers?

Reply:
yes it's used for the socreboard inside vanilla core profiler.

Reply:
This seems odd here - but I guess it's fine. (Parameters *to me* imply variables that can be changed, whereas constants do not)

Reply:
Nit: RV32E has 16 registers. It’s not inconceivable that vanilla core would support RV32E as an extension